Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -900,9 +900,10 @@
   parent: VendingMachineSeedsUnlocked
   id: VendingMachineSeeds
   suffix: Hyroponics
-  components:
```

Comment:
Why are we removing this?

Reply:
Its taking it from the parent so I didnt remove it, also its a copy of the machine so "NoSpawn" so only one exists but mapped one wont be removed or damaged.

Reply:
gotcha